scripts/crosstool-NG.sh.in

```diff
@@ -227,7 +227,7 @@ CT_SYS_REVISION=$(uname -r)
 CT_SYS_OS=$(uname -s)
 CT_SYS_MACHINE=$(uname -m)
 CT_SYS_PROCESSOR=$(uname -p)
-CT_SYS_GCC=$(gcc -dumpversion)
+CT_SYS_GCC=$(${CT_BUILD_PREFIX}gcc${CT_BUILD_SUFFIX} -dumpversion)
```

Comment:
Just curious, did you find anywhere where it is used? It looks like it is purely informational.